components/gitpod-db/go/cost_center_test.go

```diff
@@ -181,40 +172,6 @@ func TestCostCenterManager_UpdateCostCenter(t *testing.T) {
 		require.Equal(t, codes.InvalidArgument, status.Code(err))
 	})
 
-	t.Run("individual user on Other billing strategy can change spending limit of 500", func(t *testing.T) {
-		mnr := db.NewCostCenterManager(conn, limits)
-		userAttributionID := db.NewUserAttributionID(uuid.New().String())
-		cleanUp(t, conn, userAttributionID)
-
-		newCC, err := mnr.UpdateCostCenter(context.Background(), db.CostCenter{
-			ID:              userAttributionID,
-			BillingStrategy: db.CostCenter_Other,
-			SpendingLimit:   501,
-		})
-		require.NoError(t, err)
-		require.Equal(t, int32(501), newCC.SpendingLimit)
-
-	})
-
-	t.Run("team on Other billing strategy get a spending limit of 0", func(t *testing.T) {
```

Comment:
removed this because it is handled by another test that checks the new free_credits logic below